### ISSUE TITLE
add something on measuring DEI 

### ISSUE THREAD

**Issue by EstherPlomp** [alan-turing-institute/the-turing-way#2933]
### Summary

I think it would be helpful to add some resources on how to measure DEI (Diversity, Equity and Inclusion) efforts and how to facilitate improvement at the research group/institute level. 

I have no idea where this would go though!

### What needs to be done?

- [ ] Decide if this is useful
- [ ] If it is useful, where should it go?

### Who can help?

* Anyone interested!

### Resources

https://hbr.org/2021/05/how-to-measure-inclusion-in-the-workplace: 
These are the statements that form the basis of the Gartner Inclusion Index:
1.	Fair treatment: Employees at my organization who help the organization achieve its strategic objectives are rewarded and recognized fairly.
2.	Integrating differences: Employees at my organization respect and value each other’s opinions.
3.	Decision making: Members of my team fairly consider ideas and suggestions offered by other team members.
4.	Psychological safety: I feel welcome to express my true feelings at work.
5.	Trust: Communication we receive from the organization is honest and open.
6.	Belonging: People in my organization care about me.
7.	Diversity: Managers at my organization are as diverse as the broader workforce.
To put it simply: The greater the degree to which employees agree with these statements, the more inclusive the organization

https://associationsnow.com/2020/12/four-ways-to-meaningfully-measure-your-dei-efforts/ 

https://www.aleria.tech/

Cronin et al. 2021: Comprehensive examples of these documents exist and can serve as models [80], and many
laboratories host public-facing versions of these or similar values statements on their websites (Supplementary Table 1). (& following paragraph in blue)

---

### Updates

Just started discussing :)

**Comment by malvikasharan** [comment]
This is great - and I would like to support this work, especially drawing our own approaches from The Turing Way. We had submitted a proposal to Athena Awards, which was not funded but received very encouraging feedback and an invitation for reapplication. We can publish part of this as appropriate as a case study: https://docs.google.com/document/u/1/d/1IOjO1VmM20-27UPbqVTZBSbSqYXv7-k_Rbdho754d4U/edit

Also linking issue for the Guide for Accessibility:
#2730

**Comment by EstherPlomp** [comment]
Ah cool! Not that you didn't get it, but I'm looking forward to reading this!

**Comment by malvikasharan** [comment]
Tagging @rowlandm's materials on Practical Diversity - he might want to help develop this: https://www.practicaldiversity.org/

**Comment by rowlandm** [comment]
Yes, that sounds super interesting!

Just one way would be to use a simple Intersectionality Spectrum to get a feel for diversity:

https://www.practicaldiversity.org/2021/06/19/using-the-intersectionality-spectrum-to-summarise-diversity-in-senior-management/

That article is a bit old but still relevant.

**Comment by aleesteele** [comment]
This is great, thanks so much @EstherPlomp! I'm keen to support where I can.

Adding a few more things here too from the CHAOSS Folks:
- https://chaoss.community/about-chaoss/
- https://allinopensource.org/
